perf/istio-install/telemetry/metadata_exchange_filter.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: metadata-exchange
+spec:
+  configPatches:
+    - applyTo: HTTP_FILTER
+      match:
+        context: ANY # inbound, outbound, and gateway
```

Comment:
Why check this in here? we have this duplicated in many places, can we rely on some source of truth so it doesn't fall out of sync?

Reply:
other places in tools are referring to the url at istio/istio test or istio-proxy, at least we should make sure we are consistent to use the same source?